components/raftstore/src/store/peer_storage.rs

```diff
@@ -650,8 +681,13 @@ where
         low: u64,
         high: u64,
         max_size: impl Into<Option<u64>>,
+        async_to: Option<u64>,
```

Comment:
which code calls entries with non-empty async_to. Cannot find the caller using async_to.

Reply:
in `raft-rs`, you'd better review tikv/raft-rs#459 first